### PR TITLE
Create scorecard-openssf.yml

### PR DESCRIPTION
**Pull request by greenscientist**
Add a scan for the openssf scorecards (securityscorecards.dev) Gives us an overview of our security stance and can also warn if we add some unsecure deps

### CODE REVIEW

File: .github/workflows/scorecard-openssf.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '22 17 * * 4'
```
Comment (tahini) [inline .github/workflows/scorecard-openssf.yml:13]: Could you comment what this means? I always need to lookup cron syntax, I don't use it enough to remember: at 22h17 the 4th of each month? every thursday?

**Review by tahini — CHANGES_REQUESTED** [review]
Is this something like FOSSA? Our project will be listed in some third-party site investigating our code? If so, could you add some info to the commit message: How do we get to see the results of this once run? And will visitors see the run result directly from github? If so, how?

File: .github/workflows/scorecard-openssf.yml (hunk 1 above)
Comment (greenscientist) [reply .github/workflows/scorecard-openssf.yml:13]: minute         0-59
              hour           0-23
              day of month   0-31
              month          0-12 (or names, see below)
              day of week    0-7 (0 or 7 is Sun, or use names)

So this is every wednesday at 17h22

**Comment by greenscientist** [comment]
Just as a general comment, this is the default file that is generated by the tool. I'll add more in the commit message.

**Comment by kaligrafy** [comment]
Any update on this?

**Comment by greenscientist** [comment]
I should update this!

**Review by tahini — APPROVED** [review]

**Pull request merged**